sdk/tables/data-tables/tests.yml

```diff
@@ -7,7 +7,4 @@ extends:
       ServiceDirectory: tables
       MatrixFilters:
         - DependencyVersion=^$
-      EnvVars:
-        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
-        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+      SupportedClouds: 'Public,UsGov,China'
```

Comment:
Is this change intentional to test data-tables in UsGov and China environments? I don't see it present in the current `tests.yml` so I was curious why we need to add this?

Reply:
yeah not related to this PR, I will just keep Public.